test_tululbot.py

```diff
@@ -227,3 +227,52 @@ def test_who_command(app):
     assert json_response['text'] == expected_text
     assert json_response['disable_web_page_preview'] == 'false'
     assert json_response['reply_to_message_id'] == 100
+
+def test_quote(app):
+    payload = {
+        'update_id': 12345,
+        'message': {
+            'message_id': 100,
+            'text': '/quote',
+            'chat': {
+                'id': 1
+            }
+        }
+    }
+    rv = app.post('/', data=json.dumps(payload),
+                  content_type='application/json')
+    assert rv.status_code == 200
+    json_response = json.loads(rv.get_data(as_text=True))
+    assert json_response['method'] == 'sendMessage'
+    assert json_response['chat_id'] == 1
+    # Still don't know how to test
+    # assert json_response['text'] == expected_text
+    assert json_response['disable_web_page_preview'] == 'false'
+    assert json_response['reply_to_message_id'] == 100
+
+def test_who_command(app):
```

Comment:
Ah my bad. I duplicated code. How come method with the same signature is not marked as error?